docs/user_guide/installation.rst

```diff
@@ -103,32 +165,40 @@ Ensure that the path to the Homebrew packages takes priority over system
 packages in your ``$PATH``. Run ``brew --prefix`` to determine where Homebrew
 installs packages on your machine.
 
-Finally, to clone and install SiliconCompiler, run the following.
+**Install SiliconCompiler**
+
+Finally, to clone and install SiliconCompiler, run the following:
 
 .. code-block:: bash
 
    git clone https://github.com/siliconcompiler/siliconcompiler
    cd siliconcompiler
    pip install -r requirements.txt
    python -m pip install -e .
+   export SCPATH=<the full path for your siliconcompiler/siliconcompiler directory>
 
-Offline Install (Linux only)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-We also provide packages that bundle SC with all of its Python dependencies to enable installation on machines without an external internet connection. They can be found under the "Artifacts" section of any passing nightly or release build on our `builds page <https://github.com/siliconcompiler/siliconcompiler/actions/workflows/wheels.yml>`_. The packages are named ``scdeps-<pyversion>.tar.gz``, depending on which Python version they are associated with.
 
-To install from a bundle, create a Python virtual environment following the instructions above, then perform the following commands.
+Confirm your Installation
```

Comment:
@gadfort this is an example of the breaking out a section that is common to all methods of installation. This prevents copy/pasting of instruction thus reducing the likelihood of divergence. But it also potentially makes instructions more confusing for someone who wants to just read all the way through without skipping around. Do you have any thoughts on this?